### PR TITLE
Update to reflect jdk8 Windows compilation with VS2017

### PR DESCRIPTION
**Pull request by pshipton**
See https://github.com/eclipse-openj9/openj9/pull/15233

### CODE REVIEW

File: docs/version0.33.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ The following new features and notable changes since version 0.30.0 are included
 
 OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
 
+Windows&reg; builds for Java&trade; 8 are now compiled with Microsoft&reg; Visual Studio 2017. The Visual Studio redistributable files included with the build are updated to match.
```
Comment (keithc-ca) [inline docs/version0.33.md:39]: Should this refer to "OpenJDK 8" instead of "Java 8"?

Comment (pshipton) [reply docs/version0.33.md]: It doesn't hurt so I've changed it. I don't know that we've been consistent about it one way or the other in these release notes, I copied the text from 0.24.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
jenkins doc stage

**Review by doveye — APPROVED** [review]

**Comment by doveye** [comment]
Both reviews approved, so will merge this PR.

**Pull request merged**